### DISCUSSION TITLE
Can't update the adminlist.txt

### DISCUSSION THREAD

**Discussion by calvinchan999** [lloesche/valheim-server-docker#4] [Q&A]
Hi!

Thanks for making this docker image! When I has added my id name on the adminlist.txt file, it's still not work for admin commands  e.g /ban, /kick, how can i solve this?..
![image](https://user-images.githubusercontent.com/45756810/107360225-99dd5480-6b10-11eb-9e5e-858de07eb5a0.png)

![image](https://user-images.githubusercontent.com/45756810/107360067-6f8b9700-6b10-11eb-8631-3dfe2d8a002c.png)

**Comment by lloesche** [comment]
I'm guessing it is one of the following:
- a typo in the userid
- the wrong userid
- the server can not read adminlist.txt for some reason

I added a new section in the README [Admin commands](https://github.com/lloesche/valheim-server-docker#admin-commands). Can you check if your local setup is similar to what I tested there?

**Reply by lloesche** [reply]
FYI, in the log you will see two IDs.

Example:
```
02/06/2021 15:26:34: Got session request from 76561197987220805
02/06/2021 15:26:38: Got handshake from client 76561197987220805
02/06/2021 15:26:42: VERSION check their:0.142.6  mine:0.142.6
02/06/2021 15:26:42: Server: New peer connected,sending global keys
02/06/2021 15:27:02: Got character ZDOID from Luu : 774193067:1
```

The ID that must be used in adminlist.txt is the one from the line saying `Got session request` or `Got handshake` not the one from `Got character ZDOID`

Let me know if that solves it for you.

**Reply by calvinchan999** [reply]
solved, thanks
